creator-node/test/StateMonitoringManager.test.js

```diff
@@ -334,9 +334,10 @@ describe('test StateMonitoringManager initialization, events, and re-enqueuing',
     )
 
     // Verify that the queue has the correct initial job in it
-    expect(queueAdd).to.have.been.calledOnceWithExactly({
+    expect(queueAdd).to.have.been.calledOnceWith({
```

Comment:
whats diff btwn calledWith and calledWithExactly? i checked briefly and didn't find anything